source/introduction/what-is-ethereum.rst

```diff
@@ -1,42 +1,74 @@
+Ethereum is a blockchain platform that lets you build decentralized applications
+and deploy them via its distributed network of database nodes.  People around the
+world develop and maintain its open source, on-chain programming environment.
 
 ================================================================================
-A next generation blockchain
+Origins
 ================================================================================
 
-Blockchain technology is the technological basis of Bitcoin, first described by its mysterious author Satoshi Nakamoto in his white paper "Bitcoin: A Peer-to-Peer Electronic Cash System", published in 2008. While the use of blockchains for more general uses was already discussed in the original paper, it was not until a few years later that blockchain technology emerged as a generic term. A blockchain is a distributed computing architecture where every network node executes and records the same transactions, which are grouped into blocks. Only one block can be added at a time, and every block contains a mathematical proof that verifies that it follows in sequence from the previous block. In this way, the blockchain’s "distributed database" is kept in consensus across the whole network. Individual user interactions with the ledger (transactions) are secured by strong cryptography. Nodes that maintain and verify the network are incentivized by mathematically enforced economic incentives coded into the protocol.
-
-In Bitcoin's case the distributed database is conceived of as a table of account balances, a ledger, and transactions are transfers of the bitcoin token to facilitate trustless finance between individuals. But as bitcoin began attracting greater attention from developers and technologists, novel projects began to use the bitcoin network for purposes other than transfers of value tokens. Many of these took the form of "alt coins" - separate blockchains with cryptocurrencies of their own which improved on the original bitcoin protocol to add new features or capabilities. In late 2013, Ethereum's inventor Vitalik Buterin proposed that a single blockchain with the capability to be reprogrammed to perform any arbitrarily complex computation could subsume these many other projects.
-
-In 2014, Ethereum founders Vitalik Buterin, Gavin Wood and Jeffrey Wilcke began work on a next-generation blockchain that had the ambitions to implement a general, fully trustless smart contract platform.
+In 2014, Ethereum founders Vitalik Buterin, Gavin Wood and Jeffrey Wilcke began
```

Comment:
The removed text imho is a massive paragraph that tries to explain everything and is better handled elsewhere, for example with links to cryptography and bitcoin Youtubes and white papers.   

I did inadvertently take  out the "In late 2013, Ethereum's inventor Vitalik Buterin proposed " and will put it back in.